charts/internal/metrics/values.yaml

```diff
@@ -17,6 +17,7 @@ grafana-agent:
       scrape_timeout: 10s
       scrape_body_size_limit: 50MB
       scrape_cadvisor_action: keep
+      host_filter: "true"
```

Comment:
I believe we should set this to "false" by default, and add a line in `examples/stack/values-xl.yaml` which overrides the value to "true", as that example config uses daemonset mode.

I believe that setting this to true in deployment mode (the default) would mean that any nodes not running a grafana-agent pod will not have metrics scraped.

Reply:
Ah, okay, I knew I shoulda figured out the root of the issue. I was looking at this against how we had things previously set with Kustomize and assumed it should be defaulted. I was thinking last night after I left the office that I shoulda figured out the actual usage of that before just assuming.

Yes, we are running this in Daemonset mode. So that would make sense then why we would need it but in a deployment you wouldn't want it.